src/series/multi.js

```diff
@@ -9,7 +9,7 @@ import dataJoinUtil from '../util/dataJoin';
 // overriding where the series value is stored on the node (__series__) and
 // forcing the node datum (__data__) to be the user supplied data (via mapping).
 
-export default function() {
+export default function(className) {
```

Comment:
Is there any reason for having classname as an immutable property of the multi component? If not, I'd just use the standard property pattern.